src/hammer-vlsi/hammer_vlsi/hammer_tool.py

```diff
@@ -914,6 +914,49 @@ def get_bumps(self) -> Optional[BumpsDefinition]:
             pitch=self.get_setting("vlsi.inputs.bumps.pitch"),
             cell=self.get_setting("vlsi.inputs.bumps.cell"), assignments=assignments)
 
+    def get_pin_assignments(self) -> List[PinAssignment]:
+        """
+        Get a list of pin assignments in accordance with settings in the Hammer IR.
+        :return: A potentially empty list of PinAssigments.
+        """
+        pin_mode = str(self.get_setting("vlsi.inputs.pin_mode"))  # type: str
+        if pin_mode == "none":
+            return []
+        elif pin_mode != "generated":
```

Comment:
Most other places in the code we do something like:
```suggestion
        elif pin_mode == "generated":
            pass
        else:
```
I'm not super picky though. Up to you if you want to change it.

Reply:
lol sure whatever floats your python style boat.